templates/solutions/telco/index.html

```diff
@@ -618,13 +618,14 @@ <h3 class="p-heading--5">Contact us</h3>
               Canonical provides a full stack for your telecom infrastructure. To learn more about our telco solutions, get in touch.
             </p>
             <hr class="p-rule--muted" />
-            <a class="p-button" href="https://ubuntu.com/telco#get-in-touch">Contact us</a>
+            <a class="p-button" href="/contact-us" aria-controls="solutions-telco-modal">Contact us</a>
           </div>
         </div>
       </div>
     </div>
   </section>
 
   <script src="{{ versioned_static('js/tabbed-content.js') }}"></script>
+  <script defer src="{{ versioned_static('js/modals.js') }}"></script>
```

Comment:
Form template also has to be imported in the respective files
```suggestion
  <script defer src="{{ versioned_static('js/modals.js') }}"></script>
  {% include "/shared/forms/form-template.html" %}
```

Reply:
Thanks for the suggestion - that solved the issue indeed. :)